README.md

```diff
@@ -29,23 +30,28 @@ This script will download the correct binary from the project site.
 
 ### Compiling
 
- - You can compile automatically for Linux, macOS and FreeBSD with the `compile.sh` script. The script is described below.
- - You can also compile manually. See [COMPILING.md][4].
+- You can compile automatically for Linux, macOS and FreeBSD with the `compile.sh` script. The script is described below.
+- You can also compile manually. See [COMPILING.md][4].
 
 Compiling may provide better performance (1.5-3x as fast) and it supports more operating systems.
 
 #### The compile.sh script
 
 This script downloads the source code and compiles it. The script is smart enough to notify you of missing dependencies and instructing you on how to install them. The script doesn't work for Windows.
 
+Using curl:
+
+    sh -c "$(curl -sSfL -o - https://compile.cuberite.org)"
+
+Or using wget:
+
     sh -c "$(wget -O - https://compile.cuberite.org)"
```

Comment:
Is there any benefit from giving two options?  Especially seeing as the easyinstall.sh instructions only give curl.

Reply:
On macOS wget isn’t installed by default, the opposite is true for some BSD’s and Linux distros.
The easyinstall script itself uses curl, so a wget command would be useless.

Reply:
Fair enough.